packages/gasket-plugin-cypress/lib/index.js

```diff
@@ -0,0 +1,55 @@
+const { devDependencies, name } = require('../package.json');
+
+module.exports = {
+  name,
+  hooks: {
+    create: {
+      timing: {
+        last: true,
+        before: ['@gasket/plugin-lint']
+      },
+
+      handler: async function create(gasket, { files, pkg }) {
+        const generatorDir = `${__dirname}/../generator`;
+        const isReactProject = pkg.has('dependencies', 'react');
+
+        pkg.add('devDependencies', {
+          cypress: devDependencies.cypress
+        });
+
+        if (isReactProject) {
+          files.add(`${generatorDir}/*`, `${generatorDir}/**/*`);
+
+          pkg.add('devDependencies', {
+            'start-server-and-test': devDependencies['start-server-and-test']
+          });
+        }
+
+        pkg.add('scripts', {
+          'cypress': 'cypress open',
+          'cypress:headless': 'cypress run',
+          'e2e': 'start-server-and-test start http://localhost:80 cypress',
```

Comment:
This is cool!

However, when running the `gasket start` command locally, we should specify to run under the local environment. You can do this with the `--env local` flag. That'll make the e2e scripts a bit messy, so I would suggest adding another start script for this (`"start:local": "gasket start --env local"`). Then tune up the e2e scripts to call it and use the local 8080 port.

```suggestion
          'e2e': 'start-server-and-test start:local http://localhost:8080 cypress',
```